.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question on use of dotnet sdk
+    url: https://github.com/microsoftgraph/msgraph-sdk-dotnet/discussions
+    about: Please add your question in the discussions section of the repo
+  - name: Question on use of kiota
+    url: https://github.com/microsoft/kiota/discussions
+    about: Please add your question in the discussions section of the repo
```

Comment:
Similar comment to https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/2432#discussion_r1562751027